site_root/input/resources/examples/Observation-se-001.json

```diff
@@ -0,0 +1,76 @@
+{
+  "resourceType": "Observation",
+  "id": "se-001",
```

Comment:
When ingesting into a FHIR server, having the same IDs on different resources would throw an error. Currently, the Task example has also `se-001`. So, how about `st` and `so` standing for sequencing task and sequencing observation? If agreeable, we can change this ID to `so-001` and the file name to `Observation-so-001.json`